app/src/main/AndroidManifest.xml

```diff
@@ -28,6 +28,12 @@
                 android:name="android.app.lib_name"
                 android:value="" />
         </activity>
+        <activity
```

Comment:
I agree with that. In the near future, we can think about changing the theme for this, or leaving it just like now because it's just a licenses list 😄

Reply:
I agree with that. In the near future, we can think about changing the theme for this, or leaving it just like now because it's just a licenses list 😄